app/src/main/res/drawable/ic_cart_60dp.xml

```diff
@@ -0,0 +1,7 @@
+<vector android:height="60dp" android:viewportHeight="59.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_cart_60dp` appears to be unused
 Unused resources make applications larger and slow down builds.